controller/controller.ts

```diff
@@ -1,17 +1,16 @@
-import axios from 'axios';
-import asyncRetry from 'async-retry';
-import { User } from './models';
-import { transformUser } from './utils';
+import { transformUser } from '../utils/utils';
+import { getUsers } from '../services/services';
+import { User } from '../models/models';
 
-const getUsers = async (): Promise<User[]> => {
+const getUsersController = async (): Promise<User[]> => {
   try {
-    const response = await asyncRetry(() => axios.get('https://jsonplaceholder.typicode.com/users'));
-    const transformedUsers = response.data.map(transformUser);
+    const users = await getUsers();
+    const transformedUsers = users.map(transformUser);
     return transformedUsers;
   } catch (error) {
     console.error('Error al llamar a la API:', error);
     throw error;
   }
 };
 
-export { getUsers };
+export { getUsersController };
```

Comment:
Remove this line. Export the class only.

Reply:
It's ok? 

`// >> controller\controller.ts

import { usersUtil } from '../utils/utils';
import { getUsers, getPostsByUserId } from '../services/services';
import { User, Post, TransformedUser, TransformedPosts } from '../models/models';

export class UsersController {
  constructor() {}

  async getUsers(): Promise<TransformedUser[]> {
    try {
      const users = await getUsers();
      const transformedUsers = usersUtil.transformUsers(users);

      return transformedUsers;
    } catch (error) {
      console.error('Error calling API:', error);
      throw error;
    }
  }

  async getUserPosts(userId: string): Promise<TransformedPosts[]> {
    try {
      const posts = await getPostsByUserId(userId);
      const transformedPosts = usersUtil.transformUserPosts(posts);

      return transformedPosts;
    } catch (error) {
      console.error('Error calling API:', error);
      throw error;
    }
  }
}`